pallets/ddc-accounts/src/lib.rs

```diff
@@ -0,0 +1,565 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#![recursion_limit = "256"]
+
+use codec::{Decode, Encode, HasCompact};
+
+use frame_support::{
+	parameter_types,
+	traits::{Currency, DefensiveSaturating, ExistenceRequirement},
+	BoundedVec, PalletId,
+};
+pub use pallet_ddc_staking::{self as ddc_staking};
+use scale_info::TypeInfo;
+use sp_runtime::{
+	traits::{AccountIdConversion, AtLeast32BitUnsigned, Saturating, Zero},
+	RuntimeDebug, SaturatedConversion,
+};
+use sp_staking::EraIndex;
+use sp_std::prelude::*;
+
+pub use pallet::*;
+
+/// The balance type of this pallet.
+pub type BalanceOf<T> =
+	<<T as Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::Balance;
+
+parameter_types! {
+	/// A limit to the number of pending unlocks an account may have in parallel.
+	pub MaxUnlockingChunks: u32 = 32;
```

Comment:
Would be nice to have a short introduction on how the pallets work as in the other Substrate pallets.

Reply:
fixed